src/components/pages/ServiceReleaseProcess/components/ServiceDeactivate.tsx

```diff
@@ -0,0 +1,184 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { PageBreadcrumb } from 'components/shared/frame/PageBreadcrumb/PageBreadcrumb'
+import {
+  Typography,
+  PageHeader,
+  Card,
+  Checkbox,
+  Button,
+  Tooltips,
+  LoadingButton,
+} from '@catena-x/portal-shared-components'
+import { useTranslation } from 'react-i18next'
+import { useLocation, useNavigate, useParams } from 'react-router-dom'
+import { Box } from '@mui/material'
+import { useEffect, useState } from 'react'
+import { error } from 'services/NotifyService'
+import {
+  ServiceDeactivateEnum,
+  useDeactivateServiceMutation,
+} from 'features/serviceManagement/apiSlice'
+import { PAGES } from 'types/Constants'
+import { getApiBase } from 'services/EnvironmentService'
+import { fetchImageWithToken } from 'services/ImageService'
+
+export default function ServiceDeactivate() {
+  const { t } = useTranslation('servicerelease')
+  const navigate = useNavigate()
+  const serviceId = useParams().serviceId
+  const [checked, setChecked] = useState(false)
+  const { state } = useLocation()
+  const [isLoading, setIsLoading] = useState(false)
+  const items: any = state
+  const service = items?.filter((item: any) => item.id === serviceId)
+  const [deactivateService] = useDeactivateServiceMutation()
+  const [deactivateCardImage, setDeactivateCardImage] = useState('')
+  const leadImageId = service?.[0]?.leadPictureId
+
+  useEffect(() => {
+    if (leadImageId) {
+      fetchImageWithToken(
+        `${getApiBase()}/api/administration/documents/${leadImageId}`
+      )
+        .then((buffer) =>
+          setDeactivateCardImage(
+            URL.createObjectURL(new Blob([buffer], { type: 'image/png' }))
+          )
+        )
+        .catch((err) => {
+          error('ERROR WHILE FETCHING IMAGE', '', err as object)
+        })
+    }
+  }, [leadImageId])
```

Comment:
don't load image by yourself - instead pass the image url directly to the Card and 
`imageLoader={fetchImageWithToken}`